src/components/related/ProductCard.jsx

```diff
@@ -1,31 +1,54 @@
 import React, { useState, useEffect } from 'react';
-import styles from './related.module.css';
+import './related.css';
 import ComparisonModal from './ComparisonModal.jsx';
-import {useProductContext} from '../../context/ProductContext.jsx';
+import {useProductContext, updateProductContext} from '../../context/ProductContext.jsx';
 import imgNotAvailable from '../../../asset/img/image-not-available.png';
 
-const ProductCard = ({relatedProduct}) =>{
-  const { info, styles } = useProductContext();
+const ProductCard = ({relatedProduct, update}) =>{
+  const { info } = useProductContext();
+  // const update = updateProductContext()
+
   const[isOpen, setIsOpen] = useState(false);
 
   const togglePop = () => {
     setIsOpen(!isOpen)
   };
 
-  const salePrice = relatedProduct.styles[0].sale_price
+  const changeCurrentProduct = () =>{
+    update(relatedProduct.info.id)
```

Comment:
Does this even work when you have update commented out on line 9? Or is it defined somewhere else?

Reply:
Nevermind I see that update is passed in as a prop.